Sources/PeripheryKit/Configuration.swift

```diff
@@ -28,6 +28,7 @@ public class Configuration: Singleton {
     var aggressive: Bool = false
     var updateCheck: Bool = true
     var diagnosisConsole: Bool = false
+    var failOnWarnings: Bool = false
```

Comment:
I think just `strict` would be a better name. The results are only printed as warnings for the Xcode formatter, so it'd be nice for this to be more general. Plus `strict` is quite common in other tools.

Reply:
Renamed to `strict`.